filament/backend/src/Platform.cpp

```diff
@@ -115,7 +119,11 @@ DefaultPlatform* DefaultPlatform::create(Backend* backend) noexcept {
             #elif defined(IOS)
                 return new PlatformVkCocoaTouch();
             #elif defined(__linux__)
-                return new PlatformVkLinux();
+                #if defined (FILAMENT_SUPPORTS_WAYLAND)
+                    return new PlatformVkLinuxWayland();
```

Comment:
So what happens when support for X11 is off but the user tries to use OpenGL instead of Vulkan?

Reply:
I hadn't got that far on testing.  Both should co-exist.